src/features/click-to-play.js

```diff
@@ -2249,13 +2249,16 @@ const updateHandlers = {
     getYoutubePreviewsEnabled: function (resp) {
         isYoutubePreviewsEnabled = resp
     },
-    updateSetting: function (resp) {
+    setYoutubePreviewsEnabled: function (resp) {
         if (!resp.messageType || resp.value === undefined) { return }
-        window.dispatchEvent(new CustomEvent(resp.messageType, { detail: resp.value }))
+        window.dispatchEvent(new OriginalCustomEvent(resp.messageType, { detail: resp.value }))
     },
     getYouTubeVideoDetails: function (resp) {
         if (!resp.status || !resp.videoURL) { return }
-        window.dispatchEvent(new CustomEvent('ddg-ctp-youTubeVideoDetails', { detail: resp }))
+        window.dispatchEvent(new OriginalCustomEvent('ddg-ctp-youTubeVideoDetails', { detail: resp }))
```

Comment:
Shouldn't we use `utils.sendMessage` here?

Reply:
This event is just internal to the CTL content script so it doesn't need proxying.